src/levelset/levelSetObject.hpp

```diff
@@ -90,7 +90,7 @@ friend class LevelSetProxyObject;
     virtual bool                                isCellInNarrowBand(long id) const;
     virtual bool                                isInNarrowBand(const std::array<double,3> &point) const;
 
-    LevelSetIntersectionStatus                  intersectSurface(long, LevelSetIntersectionMode=LevelSetIntersectionMode::FAST_FUZZY) const;
+    LevelSetIntersectionStatus                  intersectCellSurface(long, LevelSetIntersectionMode=LevelSetIntersectionMode::FAST_FUZZY) const;
```

Comment:
This is a public function, it may be used by some external application using bitpit. You need to add the function with the new name and deprecated the old one.

Reply:
The original name was bad, the new one it's not better. Since we are renaming the function, can we find a better name, for example isCellIntersected?

Reply:
Done